kge/config-default.yaml

```diff
@@ -359,6 +359,21 @@ eval:
   # Other options
   pin_memory: False
 
+entity_ranking:
+  # How to handle cases where the correct query answer has the same score as one or
+  # multiple wrong query answers, for example: Query: (s, p, ?).  (Filtered) Answers:
+  # ['a': score 10, 'b': score 10, 'c': score 10, 'd': score 11]. Correct query
+  # answer: 'a'. Possible options are:
+  # - rounded_mean_rank: Calculate the rounded mean rank. The final rank is calculated
+  #                      as 1 + #scores_higher + #answers_with_same_score // 2,
```

Comment:
write as round_up(...), use /2. Easier to read this way.

Reply:
I have addressed the review. Only here I used round_down() as this matches X//Y.